File: Programming language/CPP/Basic concepts of C++/Arrays/String.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    char str[] = "Hello";
    cout << str << endl;
    cout << str[0] << endl;
    cout << *(str + 0) << endl;

    string s = "Hello";
    s += " World";
    cout << s << endl;

    std::string s2 = "Hello";
    const char *cstr = s2.c_str();
    
}

File: Programming language/Python/Basic concepts of Python/Data types in Python/String/string.py
s = "Hello, World"

# Slicing Strings

print(s[0:5])     
print(s[7:])    
print(s[:5])     
print(s[::2])   
print(s[::-1])  

# Useful String Methods

print(len(s))
print(s.lower())
print(s.upper())
print(s.replace("World", "Python"))
print(s.strip())
print(s.find("Hello"))
print(s.split(","))
print(s.split(" "))
print(s.split("l"))
print(s.join("Python"))

s = "abc"

print(s.isdigit())
print(s.isalpha())